app/src/models/answersModel.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+const ObjectId = Schema.ObjectId;
+
+var QuestionResponse = new Schema({
+    name: {type: String, required: true, trim: true},
+    label: {type: String, required: true, trim: true},
+    level: {type: String, required: false, trim: true}
+});
+
+var AnswerResponse = new Schema({
+    value: {type: String, required: true, trim: true},
+    label: {type: String, required: false, trim: true}
```

Comment:
As will be hard to POST a response in form-content with two objects we can remove the label from now and get it from the template.